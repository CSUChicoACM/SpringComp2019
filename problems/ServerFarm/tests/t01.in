10
5 2 10 9 5 10 11 20 1 100
