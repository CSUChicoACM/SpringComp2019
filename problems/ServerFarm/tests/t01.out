5 2 10 9 11 20 1 100
